targets/drive/mobile/main.jsx

```diff
@@ -43,28 +44,48 @@ window.handleOpenURL = require('drive/mobile/lib/handleDeepLink').default(
   hashHistory
 )
 
+const getPreviousToken = async store => {
```

Comment:
In versions 1.5.3 and earlier, the cozy token was stored by cozy-client-js. But now we want to store it ourselves.
This code should stay here for a few versions and then it will be safe to delete it.